### PR TITLE
Release notes for Mimir 2.5

### PR DESCRIPTION
**Pull request by replay**
I have skipped the section "Important changes" because I think actually there weren't any.
I have not added the section "Helm chart improvements" yet because I think I'll need a bit of help with that, as I don't have the overview of the helm chart improvements.

### CODE REVIEW

**Comment by replay** [comment]
The CHANGELOG has just been cut to prepare for the next Mimir release. Please rebase `main` and eventually move the CHANGELOG entry added / updated in this PR to the top of the CHANGELOG document. Thanks!

File: docs/sources/release-notes/v2.5.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+weight: 300
+---
+
+# Grafana Mimir version 2.5 release notes
+
+Grafana Labs is excited to announce version 2.5 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
```
Comment (pracucci) [inline docs/sources/release-notes/v2.5.md:14]: Like in previous release notes, could you spend few works about each item? See the [2.4 release notes](https://grafana.com/docs/mimir/latest/release-notes/v2.4/) as a reference for the format we adopted in previous releases.

File: docs/sources/release-notes/v2.5.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Alertmanager discord support
+- TLS minimum version and cipher suites are now configurable for HTTP and gRPC servers
+- Various improvements to lower memory usage of store-gateway and ingester
+- New dashboards: "Mimir / Overview resources" and Mimir / Overview networking"
+- Support for read-write deployment mode in jsonnet, this allows users to run Mimir as only three types of services which are responsible for read, write and backend operations
```
Comment (pracucci) [inline docs/sources/release-notes/v2.5.md:20]: This is experimental for now. I would not mention in the release notes. Also, we're not used to mention the jsonnet changes in the Mimir release notes.

File: docs/sources/release-notes/v2.5.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+## Bug fixes
+
+* Flusher: Add `Overrides` as a dependency to prevent panics when starting with `-target=flusher`. [PR 3151](https://github.com/grafana/mimir/pull/3151)
+* Query-frontend: properly close gRPC streams to the query-scheduler to stop memory and goroutines leak. [PR 3302](https://github.com/grafana/mimir/pull/3302)
+* Ruler: persist evaluation delay configured in the rulegroup. [PR 3392](https://github.com/grafana/mimir/pull/3392)
+* Ring status pages: show 100% ownership as "100%", not "1e+02%". [PR 3435](https://github.com/grafana/mimir/pull/3435)
+*  Fix panics in OTLP ingest path when parse errors occur. [PR 3538](https://github.com/grafana/mimir/pull/3538)
```
Comment (pracucci) [inline docs/sources/release-notes/v2.5.md:28]:
```suggestion
* Fix panics in OTLP ingest path when parse errors occur. [PR 3538](https://github.com/grafana/mimir/pull/3538)
```

File: docs/sources/release-notes/v2.5.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+- Various improvements to lower memory usage of store-gateway and ingester
+- New dashboards: "Mimir / Overview resources" and Mimir / Overview networking"
+- Support for read-write deployment mode in jsonnet, this allows users to run Mimir as only three types of services which are responsible for read, write and backend operations
+
+## Bug fixes
+
+* Flusher: Add `Overrides` as a dependency to prevent panics when starting with `-target=flusher`. [PR 3151](https://github.com/grafana/mimir/pull/3151)
+* Query-frontend: properly close gRPC streams to the query-scheduler to stop memory and goroutines leak. [PR 3302](https://github.com/grafana/mimir/pull/3302)
+* Ruler: persist evaluation delay configured in the rulegroup. [PR 3392](https://github.com/grafana/mimir/pull/3392)
+* Ring status pages: show 100% ownership as "100%", not "1e+02%". [PR 3435](https://github.com/grafana/mimir/pull/3435)
```
Comment (pracucci) [inline docs/sources/release-notes/v2.5.md:27]: This is really a nit. I wouldn't show it.

File: docs/sources/release-notes/v2.5.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+Grafana Labs is excited to announce version 2.5 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- Alertmanager discord support
+- TLS minimum version and cipher suites are now configurable for HTTP and gRPC servers
+- Various improvements to lower memory usage of store-gateway and ingester
+- New dashboards: "Mimir / Overview resources" and Mimir / Overview networking"
```
Comment (pracucci) [inline docs/sources/release-notes/v2.5.md:19]: I would rather add a single entry like "Improved Mimir dashboards" and then share a gist of the main improvements we did. They were several.

**Comment by replay** [comment]
Since I fixed the release branch protection rules, this branch is now protected because it matches `release-*` 🤦‍♂️ 

Closing this PR, this PR will replace it: https://github.com/grafana/mimir/pull/3580

I will make sure the incorporate the feedback from this PR on the new one that replaces it.

File: docs/sources/release-notes/v2.5.md (hunk 2 above)
Comment (replay) [reply docs/sources/release-notes/v2.5.md:20]: Thx, removed https://github.com/grafana/mimir/pull/3580/commits/504a3c573e2091ddf60941d6615c66cbe05dc183

File: docs/sources/release-notes/v2.5.md (hunk 3 above)
Comment (replay) [reply docs/sources/release-notes/v2.5.md:28]: Fixed 96e80444d

File: docs/sources/release-notes/v2.5.md (hunk 4 above)
Comment (replay) [reply docs/sources/release-notes/v2.5.md:27]: removed 644e3b0c5

File: docs/sources/release-notes/v2.5.md (hunk 5 above)
Comment (replay) [reply docs/sources/release-notes/v2.5.md:19]: How about this: https://github.com/grafana/mimir/pull/3580/files#diff-fddaf7aaf5533526b861513c83bb17e342adcfbec5e0c7c36bef8e0c47e49083R25-R29

File: docs/sources/release-notes/v2.5.md (hunk 1 above)
Comment (replay) [reply docs/sources/release-notes/v2.5.md:14]: I have tried to write something up here, I also asked Jesse to take a look at it: https://github.com/grafana/mimir/pull/3580/files#diff-fddaf7aaf5533526b861513c83bb17e342adcfbec5e0c7c36bef8e0c47e49083R14-R29

**Pull request closed without merging**